prdoc/pr_2226.prdoc

```diff
@@ -0,0 +1,29 @@
+title: Validator disabling strategy in runtime
+
+doc:
+  - audience: Node Operator
+    description: |
+      On each committed offence (no matter slashable or not) the offending validator will be
+      disabled for a whole era.
+  - audience: Runtime Dev
+    description: |
+      The disabling strategy in staking pallet is no longer hardcoded but abstracted away via
+      `DisablingStrategy` trait. The trait contains a single function (make_disabling_decision) which
+      is called for each offence. The function makes a decision if (and which) validators should be
+      disabled. A default implementation is provided - `UpToThresholdDisablingStrategy`. It
+      will be used on Kusama and Polkadot. In nutshell `UpToThresholdDisablingStrategy`
+      disables offenders up to the configured threshold (1/3 of the validators in the active set by
```

Comment:
```suggestion
      disables offenders up to the configured threshold
```

I'd remove this info from the PRdoc since it's a configuration detail, not implementation.

Reply:
Done, I've applied it manually so that I can reformat the file.